### PR TITLE
Docs: Add a link to sphinx-extensions

### PR DESCRIPTION
**Pull request by benjaoming**
Subject: Hi :wave:  I was thinking that [sphinx-extensions](https://sphinx-extensions.readthedocs.io/en/latest/) is a nice list that deserves a link.

### Feature or Bugfix

nay

### Purpose

The Awesome List has many outdated projects, so as a resource it's good but for some people a much smaller list might be a better place to start.

### Detail

nay

### Relates

also nay

### CODE REVIEW

File: doc/usage/extensions/index.rst [hunk 1]
```diff
@@ -50,14 +50,15 @@ You can find several extensions contributed by users in the `sphinx-contrib`__
 organization. If you wish to include your extension in this organization,
 simply follow the instructions provided in the `github-administration`__
 project. This is optional and there are several extensions hosted elsewhere.
-The `awesome-sphinxdoc`__ project contains a curated list of Sphinx packages,
+The `awesome-sphinxdoc`__ and `sphinx-extensions`__ projects are both curated lists of Sphinx packages,
```
Comment (jfbu) [inline doc/usage/extensions/index.rst]: linting complained about line too long can you add a line break at at most 85 characters

**Review by jfbu — COMMENTED** [review]
LGTM with nits

File: doc/usage/extensions/index.rst [hunk 2]
```diff
@@ -50,14 +50,15 @@ You can find several extensions contributed by users in the `sphinx-contrib`__
...
 and many packages use the `Framework :: Sphinx :: Extension`__ and
 `Framework :: Sphinx :: Theme`__ trove classifiers for Sphinx extensions and
 themes, respectively.
```
Comment (benjaoming) [inline doc/usage/extensions/index.rst]:
```suggestion
The `awesome-sphinxdoc`__ and `sphinx-extensions`__ projects are both curated
lists of Sphinx packages, and many packages use the
`Framework :: Sphinx :: Extension`__ and
`Framework :: Sphinx :: Theme`__ trove classifiers for Sphinx extensions and
themes, respectively.
```

**Comment by jfbu** [comment]
Thanks, merged.

**Comment by jfbu** [comment]
@benjaoming sorry that the commit message after squash and merge lost your initial one line explanation, I got fooled by github mobile showing to me the standard git cumulative starred list when squashing, but not retaining it in the end, to my surprise. Slow learner here :smile:

**Comment by benjaoming** [comment]
@jfbu thanks for being so careful with the detail and saying so. It's no harm at all... I'm unfortunately used to Open Source projects where maintainers just copy your contributions into their own commits and never say thanks :smile:

**Pull request merged**